src/main/resources/assets/scripts/js-enabled.js

```diff
@@ -0,0 +1 @@
+document.documentElement.className = ((document.documentElement.className) ? document.documentElement.className + ' js-enabled' : 'js-enabled');
```

Comment:
I know this is what was there before, but can we take the opportunity to clean this up to the much simpler:
```javascript
document.documentElement.classList.add('js-enabled');
```

Reply:
Actually, it turns out `classList` isn't supported in IE <10. Are those versions on our support list?

Reply:
No, but it's possible that it could be used to access this app.  Might be worth leaving it for now but that is a nice change